database/dataGen/genDataV02.js

```diff
@@ -0,0 +1,81 @@
+const faker = require('faker');
+
+const fs = require('fs');
+
+const generateItemData = () => {
+  const wsItem = fs.createWriteStream('/Users/yerxiong/code/yer-space/HRR43/team-leia/SDC/sdcData/items_v02.csv');
+
+  // eslint-disable-next-line quotes
+  wsItem.write(`itemId,itemName,itemPrice,itemPicture,itemFreeShipping,sellerId,sellerPicture,sellerName,sellerCountry,sellerTotalSales,sellerJoinDate,sellerStarRating,sellerReviewCount\n`);
+
+
+  // 2.5M sellers
+  let i = 2500000;
+
+  // item id
+  let k = 1;
+
+  // shipping options
+  const shipping = ['yes', 'eligible', ''];
+
+  const generateItem = () => {
+    let ok = true;
+
+    do {
+      const generateSellerInfo = {
+        sellerId: i,
+        sellerPicture: faker.image.avatar(),
+        sellerName: faker.internet.userName(),
+        sellerCountry: faker.address.country().replace(/,/g, ''),
+        sellerTotalSales: Math.floor(Math.random() * 500),
+        sellerJoinDate: Math.floor(Math.random() * (2020 - 2005 + 1)) + 2005,
+        sellerStarRating: Math.floor(Math.random() * 5),
+        sellerReviewCount: Math.floor(Math.random() * 500),
+      };
+
+      // decrement num sellers
+      i -= 1;
+
+      let j = 0;
+
+      // there will be a minimum of 10 items
+      const numOfItems = Math.floor(Math.random() * 10) + 10;
+
+      // iterate through the number of items
+      while (j < numOfItems) {
+        const generatedItemInfo = {};
+
+        Object.entries(generateSellerInfo).forEach(([key, value]) => {
+          generatedItemInfo[key] = value;
+        });
+
+        const randomImgUrl = Math.floor(Math.random() * 50) + 1;
+
+        generatedItemInfo.itemId = k;
+        generatedItemInfo.itemName = faker.commerce.productName();
+        generatedItemInfo.itemPrice = (Math.random() * 1200).toFixed(2) + 5;
```

Comment:
If this is fixed to 2 decimal places, why are the prices in the image going to 3 decimal places?

Reply:
I don't know why that's happening. I checked my csv and it goes to 3 decimal places. I'll do a fix in the frontend to display up to 2 decimal places.